packages/wrangler/src/__tests__/cloudchamber/create.test.ts

```diff
@@ -171,7 +171,7 @@ describe("cloudchamber create", () => {
 		await expect(
 			runWrangler("cloudchamber create")
 		).rejects.toThrowErrorMatchingInlineSnapshot(
-			`"image is required but it's not passed as an argument"`
+			`[Error: image is required but it's not passed as an argument]`
```

Comment:
This is the first of many snapshot format changes. Unfortunately a change was required, since Jest formats errors in `toThrowErrorMatchingInlineSnapshot()` and `toMatchInlineSnapshot()` differently to each other (while Vitest formats them the same, and doesn't allow this behaviour to be customised)